<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuideMeClass</class>
 <widget class="QMainWindow" name="GuideMeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuideMe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="openSecond">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Search on map</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openTraverse">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Traverse graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openUpdate">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Update graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mainBack">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="traverse">
     <widget class="QPushButton" name="dfsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>DFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="completeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COMPLETENESS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bfsButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BFS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back4">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>300</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="mapPathsAlgo">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="allNodes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lowest_price_map">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>third</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>300</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="mapPaths">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>381</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="update_graph">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>361</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>GuidMe</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Budget:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="budgetLine_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sourceBox_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="destBox_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Transportation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="transLine">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QLabel" name="updateLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="find">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>225</y>
        <width>361</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>welcome second</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openThird">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Paths with lowest price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>310</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>361</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>GuidMe</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Budget:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Source:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="budgetLine">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sourceBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="destBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="openThird_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>All paths</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
